### ISSUE TITLE
IP Issue

### ISSUE THREAD

**Issue by Effi3** [OatmealDome/Splat-AIO-2#6]
Whenever I enter my IP it doesn't work (It says I have entered an Invalid IP adress)

**Comment by igromanru** [comment]
Yeah, it need an improvement. Get sure you have no white spaces in the field.
